libiocage/lib/Jails.py

```diff
@@ -23,9 +23,11 @@ def __init__(self,
                  logger=None,
                  zfs=None):
 
-        libiocage.lib.helpers.init_logger(self, logger)
-        libiocage.lib.helpers.init_zfs(self, zfs)
-        libiocage.lib.helpers.init_host(self, host)
+        self.logger = libiocage.lib.helpers.init_logger(logger)
+        self.host = libiocage.lib.helpers.init_host(host, self.logger)
+
+        # this could be wrong..er.. now?:
```

Comment:
What's this comment for? Do you suspect something to be wrong with the following lines?

Reply:
so, we init_zfs(), and then we run the equivalent of the get_zfs() function, and if that's meant to be happening consistently, we may wanna just always do that in init_zfs()